yarn-project/sequencer-client/src/publisher/l1-publisher.ts

```diff
@@ -129,22 +147,60 @@ export class L1Publisher implements L2BlockReceiver {
    * @returns True once the tx has been confirmed and is successful, false on revert or interrupt, blocks otherwise.
    */
   public async processL2Block(block: L2Block): Promise<boolean> {
-    const txData = {
-      header: block.header.toBuffer(),
-      archive: block.archive.root.toBuffer(),
-      body: block.bodyToBuffer(),
-      proof: Buffer.alloc(0),
-    };
+    // TODO: Remove this block number check, it's here because we don't currently have proper genesis state on the contract
```

Comment:
Should there be a issue here so we can find it again and don't lose it?

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4118/commits/0354870e4e142090a2ef565cb6ac00f1714fa087